modules/api/php/endpoints/candidate/visit/instrument/instrument.class.inc

```diff
@@ -92,7 +92,15 @@ class Instrument extends Endpoint implements \LORIS\Middleware\ETagCalculator
     {
         $pathparts = $request->getAttribute('pathparts');
         $flags     = array_search('flags', $pathparts) !== false;
+        $user      = $request->getAttribute('user');
 
+        if ($this->_instrument->_hasAccess($user)) {
+            return new \LORIS\Http\Response\JSON\Forbidden(
+                'Can not update instruments for user' . $user
+            );
+        }
+
```

Comment:
This is to respond to all comments about removing the TODOs about permission checking in this file and the sub-endpoint `modules/api/php/endpoints/candidate/visit/instrument/flags.class.inc`